src/components/molecules/AssetStatus.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ReactElement, useEffect, useState } from 'react'
+import Status from '../atoms/Status'
+import Tooltip from '../atoms/Tooltip'
+import styles from './AssetStatus.module.css'
+
+export default function AssetStatus({
+  isDisable
+}: {
+  isDisable?: boolean
+}): ReactElement {
+  const [status, setStatus] = useState('')
+
+  useEffect(() => {
+    isDisable === true && setStatus('Disabled')
+  }, [isDisable])
+
+  return isDisable ? (
+    <div className={styles.wrapper}>
+      <Status className={styles.status} state="error" />
+      <span className={styles.text}>{status}</span>
```

Comment:
so what's going to be the default `status` if `isDisable` is omitted as its's marked as optional?